### PR TITLE
new ci user

### PR DESCRIPTION
**Pull request by mapsam**
This the travis/appveyor configuration to use env settings instead of encrypted tokens. Tokens have been rotated.

cc @mapbox/core-tech

### CODE REVIEW

**Comment by mapsam** [comment]
Test package `https://mapbox-node-binary.s3.amazonaws.com/zipfile/v0.5.11-testciusermapsam/node-v51-linux-x64.tar.gz` was published 👍 - this is ready to merge whenever someone can give it a review!

File: package.json [hunk 1]
```diff
@@ -1,43 +1,51 @@
...
-  "version"       : "0.5.11",
-  "main"          : "./lib/index.js",
-  "description"   : "C++ library for handling zipfiles in node",
-  "keywords"      : ["zipfile", "uncompress", "unzip", "zlib"],
-  "url"           : "http://github.com/mapbox/node-zipfile",
-  "repository" : {
-           "type" : "git",
-           "url"  : "git://github.com/mapbox/node-zipfile.git"
+  "name": "zipfile",
+  "version": "0.5.11-testciusermapsam",
```
Comment (GretaCB) [inline package.json]: dev version?

**Comment by GretaCB** [comment]
@mapsam just added a comment. I had a momentary ponder whether it was worth it to update deps as part of this, but that could get hairy and probably best as a separate PR.

File: package.json (hunk 1 above)
Comment (mapsam) [reply package.json]: oops, just need to undo this - was testing out the new creds. Thanks!

**Pull request merged**